pkg/nsx/services/ipblocksinfo/store.go

```diff
@@ -96,3 +98,37 @@ func (vs *VPCStore) Apply(i interface{}) error {
 	}
 	return nil
 }
+
+type VpcAttachmentStore struct {
```

Comment:
Question: what are the differences between this `VpcAttachmentStore` and the other under package `vpc`? Why objects in this store uses "path" as key, and the other uses "vpcID_default"?

Reply:
as vpc may have more than one attachment.